rflx/generator/session.py

```diff
@@ -675,12 +681,14 @@ def _create_uninitialized_function(
         )
 
     def _create_global_initialized_function(
-        self, composite_globals: Sequence[decl.VariableDeclaration], is_global: Callable[[ID], bool]
+        self,
+        composite_globals: Sequence[decl.VariableDeclaration],
+        is_global: Callable[[ID], bool],
     ) -> UnitPart:
         if not composite_globals:
             return UnitPart()
 
-        self._session_context.used_types.append(const.TYPES_INDEX)
+        self._session_context.used_types.append(ID(ID(const.TYPES_INDEX)))
```

Comment:
Why is that double `ID` call necessary (more instances below)?

Reply:
Apparently I forgot to clean up this module after trying out different variants. I have removed a lot of unnecessary `ID` calls in this module and also some in other modules.